judges/quality-of-service/quality-of-service.rb

```diff
@@ -32,15 +32,26 @@
   total = 0
   success = 0
   duration = 0
+  mttr = {}
   Fbe.unmask_repos.each do |repo|
     Fbe.octo.repository_workflow_runs(repo, created: ">#{f.since.utc.iso8601[0..9]}")[:workflow_runs].each do |json|
+      workflow_id = json[:workflow_id]
+      if json[:conclusion] == 'failure'
+        mttr[workflow_id] = { last_failed_at: json[:updated_at] }
+      elsif json[:conclusion] == 'success' && mttr[workflow_id] && mttr[workflow_id][:last_failed_at]
+        mttr[workflow_id][:mttrs] ||= []
```

Comment:
@Suban05 maybe we not needed nested constructions?
Isn't it easier to record a period of time in a regular array `mttrs` ?
And this line will no longer be needed  `mttrs = mttr.values.flat_map { |data| data[:mttrs] }.compact`

```ruby
mttrs = []
....

mttrs << (json[:updated_at] - mttr[workflow_id][:last_failed_at]).to_i
```